integrations/dd-sdk-android-compose/src/main/kotlin/com/datadog/android/compose/BackstackKeyResolver.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.compose
+
+/**
+ * Resolves a stable key for an item in a backstack.
+ * A stable key means that the same item will always return the same key during its lifetime in the
+ * backstack.
```

Comment:
should we also add the statement that it shouldn't be also 2 different items with the same key?